typescript/sdk/src/gas/HyperlaneIgpDeployer.ts

```diff
@@ -126,84 +109,73 @@ export class HyperlaneIgpDeployer extends HyperlaneDeployer<
           desiredGasData.tokenExchangeRate,
         )
       ) {
-        this.logger(
-          `${chain} -> ${remote} existing gas data:\n`,
-          prettyRemoteGasData(remoteGasDataConfig),
-        );
-        this.logger(
-          `${chain} -> ${remote} desired gas data:\n`,
-          prettyRemoteGasData(desiredGasData),
-        );
-        configsToSet[gasOracleAddress].push({
+        configsToSet.push({
+          remote,
           remoteDomain: this.multiProvider.getDomainId(remote),
           ...desiredGasData,
         });
       }
     }
-    // loop through each gas oracle and batch set the remote gas data
-    for (const gasOracle of Object.keys(configsToSet)) {
-      const gasOracleContract = StorageGasOracle__factory.connect(
-        gasOracle,
-        this.multiProvider.getSigner(chain),
-      );
-      if (configsToSet[gasOracle].length > 0) {
-        await this.runIfOwner(chain, gasOracleContract, async () => {
-          this.logger(
-            `Setting gas oracle on ${gasOracle} for ${configsToSet[
-              gasOracle
-            ].map((config) => config.remoteDomain)}`,
+
+    if (configsToSet.length > 0) {
+      await this.runIfOwner(chain, gasOracle, async () => {
```

Comment:
give a warning if runIfOwner doesn't run to make the deployer is aware there config didn't get applied

Reply:
isnt that what the logs in `runIfOwner` are for?